### PR TITLE
Add edit attributes + product models

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/product-mass-actions.md [hunk 1]
```diff
@@ -68,12 +68,39 @@ After some of the bulk actions the rules will be automatically applied.
...
+1.  Search for products in the grid with the appropriate filters
+2.  Select the products to edit: tick the products you want to add via the down arrow of the toolbar: `All` for all the products recorded, `All visible` for all visible products on the ongoing page, `none` for none of the products
+3.  Click on `Bulk Actions`
+4.  Select `Edit attributes` and then `Next`
+5.  Click on `Select attributes` to select the attributes you want to update (for example the collection and the brand)
+6.  Add values for the selected attributes
+![Mass edit options](../img/Products_BulkActionEditAttribute.png)
+7.  Click on `Next`
+8.  Confirm the action with the `Confirm` button located in the middle of the screen
+9.  The attributes values will be updated on the products selected
```
Comment (NolwennP) [inline content/md/products-super-power/product-mass-actions.md]: selected products

File: content/md/products-super-power/product-mass-actions.md [hunk 2]
```diff
@@ -68,12 +68,39 @@ After some of the bulk actions the rules will be automatically applied.
...
+
+:::warning
+This action doesn't add attribute to products, only the attributes belonging to the families of the selected products will be edited.
+:::
+
+:::info
+The changes will be done for the scope and the locale selected in the product’s grid. If you have several locales or scopes, you will have to do several bulk actions.
```
Comment (NolwennP) [inline content/md/products-super-power/product-mass-actions.md]: product grid

File: content/md/products-super-power/product-mass-actions.md [hunk 3]
```diff
@@ -68,12 +68,39 @@ After some of the bulk actions the rules will be automatically applied.
...
+:::
+
+:::tips
+If you select some product models in the grid, the attributes will be updated on the product models and all its children variant products according to the [family variant](/articles/what-about-products-variants.html#what-is-a-family-variant) definition. For example, if the description (defined as `common attributes`) and the weight (defined for each variant product) is mass edited, the description will be updated on the product models and the weight on each child variant product of the product models selected.
```
Comment (NolwennP) [inline content/md/products-super-power/product-mass-actions.md]: of the selected product models.

File: content/md/products-super-power/product-mass-actions.md (hunk 1 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: je vais y arriver :)

File: content/md/products-super-power/product-mass-actions.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: oki

File: content/md/products-super-power/product-mass-actions.md (hunk 3 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: rah again

**Pull request merged**